src/components/status.less

```diff
@@ -1,3 +1,5 @@
+@import (reference) "~less/compass/palette.less";
```

Comment:
@durran I wanted to avoid hardcoding colors so i added in this line, but not sure if there is a better way to do this?

Reply:
That’s perfect.